src/main/java/core/basesyntax/service/ShopServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import java.io.File;
+
+public class ShopServiceImpl implements ShopService {
+    private ShopServiceStrategy strategy;
+    private StorageDaoImpl stDao;
+    private CSvReader csvReader;
+
+    public ShopServiceImpl(ShopServiceStrategy strategy,
+                           StorageDaoImpl stDao, CSvReader csvReader) {
+        this.strategy = strategy;
+        this.stDao = stDao;
+        this.csvReader = csvReader;
+    }
+
+    public String readAndConvert(String fromFileName) {
+        return csvReader.readFile(fromFileName);

```

Comment:
`readAndConvert` - confusing naming of method, i see any conversation logic in this method. 

lets move convertation logic from csvReader to this method.